src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -219,17 +219,49 @@ message Container {
   map<string, string> env = 7;
 
   // (Optional) Constraints that Titus will prefer to fulfill but are not
-  // required.
+  // required. These constraints apply to the whole task.
   Constraints softConstraints = 8;
 
   // (Optional) Constraints that have to be met for a task to be scheduled on
-  // an agent.
+  // an agent. These constraints apply to the whole task.
   Constraints hardConstraints = 9;
 
   // (Optional) Experimental features
   google.protobuf.Any experimental = 10;
 }
 
+// BasicContainer stores the minimal data required to declare extra containers
+// to a job. This is contrast to the Container message, which has other data
+// that are not strictly tied to the one main container. For example,
+// *resources* (ram/cpu/etc) for the whole *task* are declared in the main
+// Container message, not in a basic container.
+message BasicContainer {
+  // (Required) the Name of this container
+  string name = 1;
+
+  // (Required) Image reference.
+  Image image = 2;
+
+  // (Optional) Override the entry point of the image.
+  // If set, the command baked into the image (if any) is always ignored.
+  // Interactions between the entry point and command are the same as specified
+  // by Docker:
+  // https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact
+  // Note that, unlike the main container, very little additional processing
+  // happens on the raw entrypoint here (no string splitting).
+  repeated string entryPoint = 3;
+
+  // (Optional) Additional parameters for the entry point defined either here
+  // or provided in the container image.
+  // Note that, unlike the main container, very little additional processing
```

Comment:
`very little additional processing` makes this phrase hard to understand IMO. I think you should either just say that the spring splitting does not happen or elaborate on what the additional processing is.